lib/BiochemistryAPI/BiochemistryAPIImpl.py

```diff
@@ -173,22 +187,24 @@ def search_compounds(self, ctx, params):
         """
         # ctx is the context object
         # return variables are: out_compounds
-        #BEGIN search_compounds
+        # BEGIN search_compounds
         logging.info("Starting search_compounds")
         logging.info("Params: {}".format(params))
-        utils.check_param(params, ['query'], ['limit'])
-        limit = params.get('limit', 10)
-        normed_query = params['query'].strip().lower().translate(utils.ttable)
+        utils.check_param(params, ["query"], ["limit"])
+        limit = params.get("limit", 10)
+        normed_query = params["query"].strip().lower().translate(utils.ttable)
         matching_ids = self.comp_search_dict.get(normed_query, [])
```

Comment:
`self.comp_search_dict` is a mapping from tokens (from `utils._tokenize_string(...)`) to compound IDs. `normed_query` is an entire normalized string.

Reply:
Not sure if anything should be changed here?